packages/preview/course-poster-vuw/0.1.0/course-poster-vuw.typ

```diff
@@ -0,0 +1,348 @@
+)
+
+// details separators
+place(
+  dx: details-left,
+  dy: pageheight - details-top,
+  line(
+  length: details-width,
+  stroke: stoke-width + col,
+)
+)
+
+place(
+  dx: details-left,
+  dy: pageheight - details-top + details-height,
+  line(
+  length: details-width,
+  stroke: stoke-width + col,
+)
+)
+
+// schedule separator
+place(
+  dx: 0.5*(pagewidth - schedule-width),
+  dy: pageheight  - margin-shaded-bottom - schedule-spacer,
+  line(
+  length: schedule-width,
+  stroke: (thickness: 0.5*stoke-width , paint: col , dash: "dotted"),
+)
+)
+
+place(
+  dx: 0.5*(pagewidth - schedule-width),
+  dy: pageheight  - margin-shaded-bottom - 2*schedule-spacer,
+  line(
+  length: schedule-width,
+  stroke: (thickness: 0.5*stoke-width , paint: col , dash: "dotted"),
+)
+)
+
+}
+
+
+let vuwcol = rgb(0,71,48)
+
+
+// computed
+let coursediscipline = courseid.slice(0,4)
+let coursecode = courseid.slice(4,7)
+let year = coursecode.at(0)
+let courseurl = qrcodeurl + coursediscipline + "/" + coursecode
+// https://www.wgtn.ac.nz/courses/phys/304/2023
+
+let col = (
+  "1": rgb(235,157,12),
+  "2": rgb(0,158,224),
+  "3": rgb(86,163,38),
+  "4": rgb(226,0,122),
+).at(year)
+
+// set up page
+set page(height: pageheight, width: pagewidth, margin: 0mm, background: bkg(col: col))
+set text(size: 13pt)
+set par(justify: false, leading: 0.2em)
+show raw: set text(font: "Source Code Pro", size: 1.1em, tracking: -0.1pt)
+
+
+// VUW stuff
+
+// note that must be before defining • colour globally (bug)
+place(
+  dx: address-left,
+  dy:  pageheight - margin-shaded-bottom + address-top,
+  box(width: address-width, height: logo-height)[
+  #set align(top)
+  #set par(justify: true, leading: 0.7em)
+  // #rect(width:100%, height: 100%)
+  #set text(weight: 200, size: 11pt, fill: vuwcol)
+
+  #scaps(tracking: 0.2pt,weight:400)[#contact.school] \
+  #scaps(tracking: 0.2pt,weight:400)[#contact.faculty] \
+  #contact.university\
+  #scaps(tracking: 1pt,weight:400)[p:]  #raw(contact.phone) • #scaps(tracking: 1pt,weight:400)[e:]  #raw(contact.email) • #scaps(tracking: 1pt,weight:400)[w:]  #raw(contact.website)
+
+  ],
+)
+
+// qr code
+place(
+  dx: pagewidth - margin-body-left - logo-height,
+  dy:  pageheight - margin-shaded-bottom + address-top,
+  qrcode(courseurl, width: logo-height, ecl:"l",
+  colors: (white, vuwcol), quiet-zone: 0)
+  // image("qr.png", height: logo-height)
+)
+
+// logo
+place(
+  dx: margin-body-left,
+  dy:  pageheight - margin-shaded-bottom + address-top, 
+  box(height: logo-height, width: auto)[#logo]  
+)
+
+
+// now define it globally
+show "•": text.with(weight: "extralight", fill: col)
+show "·": text.with(weight: "extralight", fill: col)
+show "↯": text.with(fill: col)
+
+
+// image
+place(
+  dx: margin-body-left,
+  dy: image-top,
+  rect(width:image-width, height:0.5*image-width, stroke: 0.5*stoke-width)
+)
+
+place(
+  dx: margin-body-left,
+  dy:  image-top,
+  image(courseimage, width:image-width, height:0.5*image-width),
```

Comment:
The path will be resolved relative to the package, so this currently breaks after template initialization. To make this user-configurable, you'd need to move the whole image call into the user code. You can still set the image width and height here using a set rule.  You can take a look at the `dashing-dept-news` template to see how that works.

Reply:
Thanks, I've changed this.